lighthouse-core/runner.js

```diff
@@ -150,6 +157,7 @@ class Runner {
           generatedTime: (new Date()).toJSON(),
           initialUrl: opts.initialUrl,
           url: opts.url,
+          lighthouseRunWarnings,
```

Comment:
🚲 🏠  can we ditch the `lighthouse` prefix, it's the LHR so hopefully everything is lighthouse related

don't want to end up in a classic `lighthouse-core`, `lighthouse-cli`, `lighthouse-*` situation

Reply:
SG